src/graph/rg_matrix/rg_matrix_iter.c

```diff
@@ -49,7 +49,7 @@ static inline void _set_iter_range
 
 static inline void _init_iter
 (
-	GxB_Iterator *it,
```

Comment:
`_set_iter_range` needs to be well documented, broken down to GrB_Info different values
in addition add a unit test which gets us to line 42: `while (info == GrB_NO_VALUE && GxB_rowIterator_getRowIndex(it) < max_row) {` making sure the function is fully covered by unit tests.